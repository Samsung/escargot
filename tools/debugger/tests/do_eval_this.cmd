eval this
s
eval this
quit
