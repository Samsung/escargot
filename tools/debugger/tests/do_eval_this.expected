Connecting to: localhost:6501
Connection created!!!
Stopped at tools/debugger/tests/do_eval_this.js:24
(escargot-debugger) eval this
[object global](escargot-debugger) s
Stopped at tools/debugger/tests/do_eval_this.js:21 (in foo() at line:20, col:1)
(escargot-debugger) eval this
function foo() {
    foo.name
}(escargot-debugger) quit
Connection closed.
